devel/legacy-support/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        cjones051073 macports-legacy-support 55fdb329676c52cbf3d63f6e886d17d12e32ae3d
```

Comment:
Why don't we simply create a repo inside the macports org?

Also, I would be in favour of tagging the releases (even if using just the date), so that we don't need to provide long checksums.

Reply:
Whom has the rights to create a fork of my repo under the MacPorts organisation ?

Reply:
Probably one of @ryandesign @raimue @jmroot @neverpanic